packages/editor/src/components/autosave-monitor/index.js

```diff
@@ -29,15 +29,11 @@ export class AutosaveMonitor extends Component {
 	}
 
 	componentDidUpdate( prevProps ) {
-		if (
-			this.props.disableIntervalChecks &&
-			this.props.editsReference !== prevProps.editsReference
-		) {
-			this.props.autosave();
+		if ( this.props.disableIntervalChecks ) {
```

Comment:
This will break native app that uses `<AutosaveMonitor disableIntervalChecks />`, so it expects to autosave at every edit, as per above comment:
`The timer may be disabled by setting `props.disableIntervalChecks` to `false`. In that mode, any change will immediately trigger props.autosave()`

Reply:
Ohhh I get it now, I'll restore it. I thought it was used to disable autosaves.

Reply:
Can you also update the above comment that is wrong to:
```
by setting `props.disableIntervalChecks` to `true`
```

Reply:
@ntsekouras is right. This change will break the autosave for the mobile app.